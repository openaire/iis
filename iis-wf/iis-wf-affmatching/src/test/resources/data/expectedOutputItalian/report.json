{"key": "processing.docOrgMatching.docOrgReferences.affiliationBased", "type":"COUNTER", "value": "3"}
{"key": "processing.docOrgMatching.docs.affiliationBased", "type":"COUNTER", "value": "1"}