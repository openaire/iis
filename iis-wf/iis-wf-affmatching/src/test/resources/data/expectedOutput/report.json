{"key": "processing.docOrgMatching.docOrgReferences.affiliationBased", "type":"COUNTER", "value": "10"}
{"key": "processing.docOrgMatching.docs.affiliationBased", "type":"COUNTER", "value": "6"}